app/templates/ui/app/search/search-results.html

```diff
@@ -1,6 +1,6 @@
 <div ng-repeat="result in results">
   <h4>
-    <a ui-sref="root.view({uri: result.uri})">{{ result.uri }}</a>
+    <a ui-sref="root.view({uri: result.uri})">{{ result.title || result.uri }}</a>
```

Comment:
If that kind of pattern is going to be added, I would ask that it be added to the `ml-results` directive. Similar processing is already being done to add a `link` property to each result object: https://github.com/joemfb/ml-search-ng/blob/master/src/directives/ml-results.directive.js#L59